src/main/java/com/epam/prejap/ess/helpers/collections/EmptyListComparison.java

```diff
@@ -0,0 +1,192 @@
+     *      <li>an immutable collection</li>
+     *      <li>serializable</li>
+     * </ul>
+     * Implementations of this method need not create a separate List object for each call.
+     * Every call of this method return a cast reference from the EMPTY_LIST field.
+     * Using this method is likely to have comparable cost to using the like-named field.
+     * (Unlike this method, the field does not provide type safety.)
+     * <br>
+     * Since then it is recommended way to obtain an empty list.
+     *
+     * @see <a href="https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/util/Collections.html#emptyList()">Java SE 16 & JDK 16 - emptyList()</a>
+     */
+    private <T> List<T> emptyList() {
+        return Collections.emptyList();
+    }
+
+    /**
+     * Print information about both ways of obtaining an empty list from the Collections class.
+     * Inside is also included some examples of usage for each manner with a brief comment
+     */
+    public static void main(String[] args) {
+
+        System.out.println(checkIfBothListsAreEquals()
+                ? "Both lists are equals"
+                : "Both lists are not equals");
+
+        System.out.println(checkIfBothListsAreEmpty()
+                ? "Both lists are empty. Their size is equal to 0."
+                : "Both lists are not empty. Their size is not equal to 0.");
+
+        System.out.println(checkIfBothListsAreSerializable()
+                ? "Both lists are serializable"
+                : "Both lists are not serializable");
+
+        checkIfListIsImmutable(Collections.EMPTY_LIST, "Collections.EMPTY_LIST");
+
+        checkIfListIsImmutable(Collections.emptyList(), "Collections.emptyList()");
+
+        System.out.println(checkIfProvidedEmptyListIsAnInstanceOfRawList(Collections.EMPTY_LIST)
+                ? "The Collections.EMPTY_LIST is an instance of List"
+                : "The Collections.EMPTY_LIST is NOT an instance of List");
+
+        /*
+         * The code below will cause a compilation error with the specified message:
+         * "'List' cannot be safely cast to 'List<Object>'"
+         * */
+        //System.out.println(Collections.EMPTY_LIST instanceof List<Object>);
+
+        /*
+         * The code below is correct, and it will not cause any errors. Both conditions with "instanceof" are always true.
+         * */
+        System.out.println(checkIfProvidedEmptyListIsAnInstanceOfRawList(Collections.emptyList())
+                ? "The Collections.emptyList() is an instance of List"
+                : "The Collections.emptyList() is not an instance of List");
+        System.out.println(checkIfEmptyListFromCollectionsStaticMethodIsAnInstanceOfObjectsList()
+                ? "The Collections.emptyList() is an instance of List<Object>"
+                : "The Collections.emptyList() is not an instance of List<Object>");
+
+        /*
+         * The return type is a raw List
+         * */
```

Comment:
```suggestion
         */
```

I can see that in many places to be fixed. To end multiline comment you can just use `*/` without additional asterisk.